ArithmeticException (division by zero) in apoc.meta.schema/graph

## Expected Behavior (Mandatory)

--

## Actual Behavior (Mandatory)

A call of apoc.meta.schema() as well as apoc.meta.graph result in the following error:
```
Neo.ClientError.Procedure.ProcedureCallFailed
Neo.ClientError.Procedure.ProcedureCallFailed: Failed to invoke procedure `apoc.meta.schema`:
Caused by: java.lang.ArithmeticException: / by zero
```

### Steps (Mandatory)

- Download and import the Neo4j DB dump from: https://reactome.org/download/current/reactome.graphdb.tgz
- Call apoc.meta.graph 

## Specifications (Mandatory)

### Versions

  - OS: macOS 10.14.3
  - Neo4j: 3.5.3
  - Neo4j-Apoc: 3.5.0.2

--- a/src/main/java/apoc/meta/Meta.java
+++ b/src/main/java/apoc/meta/Meta.java
@@ -479,7 +479,8 @@ public class Meta {
             if (min >= max) {
                 return -1L;
             }
-            return ThreadLocalRandom.current().nextLong(min, max);
+            long randomValue = ThreadLocalRandom.current().nextLong(min, max);
+            return randomValue == 0L ? -1L : randomValue; // it can't return zero as it's used in % ops
         } else {
             return sample;
         }

